src/pull.js

```diff
@@ -0,0 +1,94 @@
+/* @flow */
+
+import printit from 'printit'
+
+import RemoteCozy from './remote/cozy'
+import Prep from './prep'
+import Pouch from './pouch'
+import * as conversion from './conversion'
+
+import type { RemoteDoc } from './remote/document'
+import type { LocalDoc } from './local/document'
+
+const log = printit({
+  prefix: 'Pull',
+  date: true
+})
+
+// Pull changed files and directories metadata from the remote Cozy, and merge
+// them into the local database.
+export default class Pull {
```

Comment:
`MetadataPull`?
I'd like to keep the `Pull` suffix since at least it indicates the direction...
Or were you think of moving this back into the `RemoteWatcher`? (I thought it was to much responsibility for a single class...)